src/main/CommandLine.cpp

```diff
@@ -159,6 +159,20 @@ fileNameParser(std::string& string)
     return requiredArgParser(string, "FILE-NAME");
 }
 
+clara::Opt
+processIDParser(uint& num)
+{
+    return clara::Opt{num, "PROCESS-ID"}["--process_id"](
+        "for spawning multiple instances in fuzzing paralellization");
+}
+
+clara::Opt
+isOverlayFuzzer(bool& overlayFuzz)
+{
+    return clara::Opt{overlayFuzz}["--overlay_fuzz"](
```

Comment:
wouldn't it be cleaner to have something like
`fuzz --mode=overlay` or `fuzz --mode=tx` ; that way we can add as many fuzzers as we want